t_reconf/test_reconf_base.py

```diff
@@ -1816,11 +1829,214 @@ class TestNegativeReconf(tester.TempestaTest):
+
+srv_group grp1 {{
+    server_error {SERVER_IP}:8000;
+}}
+vhost grp1 {{
+    proxy_pass grp1;
+}}
+http_chain {{
+    -> grp1;
+}}
+            """,
+                deproxy_id="deproxy",
+            ),
+            marks.Param(
+                name="server_h2",
+                valid_config=f"""
+listen 443 proto=h2;
+listen 80;
+cache 2;
+cache_fulfill * *;
+cache_methods GET;
+access_log off;
+
+tls_certificate {TEMPESTA_WORKDIR}/tempesta.crt;
+tls_certificate_key {TEMPESTA_WORKDIR}/tempesta.key;
+tls_match_any_server_name;
+frang_limits {{http_strict_host_checking false;}}
+
+srv_group grp1 {{
+    server {SERVER_IP}:8000;
+}}
+vhost grp1 {{
+    proxy_pass grp1;
+}}
+http_chain {{
+    -> grp1;
+}}
+            """,
+                invalid_config=f"""
+listen 443 proto=h2;
+listen 80;
+cache 2;
+cache_fulfill * *;
+cache_methods GET;
+access_log off;
+
+tls_certificate {TEMPESTA_WORKDIR}/tempesta.crt;
+tls_certificate_key {TEMPESTA_WORKDIR}/tempesta.key;
+tls_match_any_server_name;
+frang_limits {{http_strict_host_checking false;}}
+
+srv_group grp1 {{
+    server_error {SERVER_IP}:8000;
+}}
+vhost grp1 {{
+    proxy_pass grp1;
+}}
+http_chain {{
+    -> grp1;
+}}
+            """,
+                deproxy_id="deproxy_h2",
+            ),
+            marks.Param(
+                name="listen_http",
+                valid_config=f"""
+listen 443 proto=h2;
+listen 80;
+cache 2;
+cache_fulfill * *;
+cache_methods GET;
+access_log off;
+
+tls_certificate {TEMPESTA_WORKDIR}/tempesta.crt;
+tls_certificate_key {TEMPESTA_WORKDIR}/tempesta.key;
+tls_match_any_server_name;
+frang_limits {{http_strict_host_checking false;}}
+
+srv_group grp1 {{
+    server {SERVER_IP}:8000;
+}}
+vhost grp1 {{
+    proxy_pass grp1;
+}}
+http_chain {{
+    -> grp1;
+}}
+            """,
+                invalid_config=f"""
+listen 443 proto=h2;
+listen 80;
+listen aaaaa;
+cache 2;
+cache_fulfill * *;
+cache_methods GET;
+access_log off;
+
+tls_certificate {TEMPESTA_WORKDIR}/tempesta.crt;
+tls_certificate_key {TEMPESTA_WORKDIR}/tempesta.key;
+tls_match_any_server_name;
+frang_limits {{http_strict_host_checking false;}}
+
+srv_group grp1 {{
+    server_error {SERVER_IP}:8000;
+}}
+vhost grp1 {{
+    proxy_pass grp1;
+}}
+http_chain {{
+    -> grp1;
+}}
+            """,
+                deproxy_id="deproxy",
+            ),
+            marks.Param(
+                name="listen_h2",
+                valid_config=f"""
+listen 443 proto=h2;
+listen 80;
+cache 2;
+cache_fulfill * *;
+cache_methods GET;
+access_log off;
+
+tls_certificate {TEMPESTA_WORKDIR}/tempesta.crt;
+tls_certificate_key {TEMPESTA_WORKDIR}/tempesta.key;
+tls_match_any_server_name;
+frang_limits {{http_strict_host_checking false;}}
+
+srv_group grp1 {{
+    server {SERVER_IP}:8000;
+}}
+vhost grp1 {{
+    proxy_pass grp1;
+}}
+http_chain {{
+    -> grp1;
+}}
+            """,
+                invalid_config=f"""
+listen 443 proto=h2;
+listen 80;
+listen aaaaa;
```

Comment:
You add a new invalid listen, not make reconfiguration. 
```suggestion
listen aaaaa;
listen 80;
```